test/unittests/commands/excel2xml/test_excel2xml_lib.py

```diff
@@ -501,7 +500,8 @@ def test_make_text_prop(self) -> None:
             lambda: excel2xml.make_text_prop(":test", excel2xml.PropertyElement(value="a", encoding="unicode")),
         )
 
-        # encoding="utf8"
+    def test_make_text_prop_utf8(self) -> None:
+        """test encoding=utf8"""
```

Comment:
is this docstring needed?

Reply:
No, but I find it helpful

Reply:
to me it seems like a duplication of the method name. But if you prefer to have it there, sure!